curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/66507c72ee0e194fd011ff39.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 66507c72ee0e194fd011ff39
+title: Step 51
+challengeType: 1
+dashedName: step-51
+---
+
+# --description--
+
+Now that you have finished reviewing the basics of functions, you can remove the code you wrote in the last step and proceed to the next lesson.
+
+# --hints--
+
+You should not have a `addTwoNumbers` function in your code.
+
+```js
+
+assert.notMatch(code, /function\s+addTwoNumbers\s*\(/);
+
+```
+
+You should not have a `addTwoNumbers(5, 10)` function call in your code.
+
+```js
+
+assert.notMatch(code, /addTwoNumbers\(\s*5\s*,\s*10\s*\)/);
+
+```
+
+You should not have a `addTwoNumbers(10, 20)` function call in your code.
+
+```js
+
+assert.notMatch(code, /addTwoNumbers\(\s*10\s*,\s*20\s*\)/);
+
+```
```

Comment:
> assert.isAtMost(code.match(/console\.log/)?.length, 1);

please correct me if i am wrong but there is only one console log statement which has to be removed.

Reply:
There is one that needs to be removed, but there's another one further down in the code that stays. We allow for a length of 1 so that the camper does not have to remove that one.